base_layer/wallet_ffi/README.md

```diff
@@ -35,11 +35,23 @@ For macOS Mojave additional headers need to be installed, run
 ```Shell Script
 open /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg
 ```
-and follow the prompts
+and follow the prompts.
+
+For Catalina, if you get compilation errors such as these:
+
+    xcrun: error: SDK "iphoneos" cannot be located 
+    xcrun: error: unable to lookup item 'Path' in SDK 'iphoneos'
+
+Switch the XCode app defaults with:
+
+    sudo xcode-select --switch /Applications/Xcode.app
 
 ## Android Dependencies
 
-Download the [Android NDK Bundle](https://developer.android.com/ndk/downloads)
+Install [Android Studio](https://developer.android.com/studio) and then use the SDK Manager to install the Android NDK
+along with the SDK of your choice (Android Q is recommended). The command line tools are also very handy.
+
```

Comment:
When setting up an AVD (Android Virtual Device) please note that a 64-bit image (x86_64) needs to be used and not a 32-bit image (x86). This is to run the application on the simulator with these libraries.

Reply:
The NDK also needs to be installed in addition to the SDK